<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceDetectionDialog</class>
 <widget class="QDialog" name="FaceDetectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Παραμετροποίηση Εντοπισμού Προσώπων</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/argus/res/app_icons/detection.png</normaloff>:/argus/res/app_icons/detection.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>461</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin" stdset="0">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Εκκίνηση</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Κλείσιμο</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Επιλογές Αποθήκευσης</string>
   </property>
   <widget class="QCheckBox" name="saveToVideoCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Αποθήκευση στιγμιότυπων όπου εντοπίστηκαν πρόσωπα σε βίντεο.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="saveToImageCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Εξαγωγή εικόνων απο τα πρόσωπα που εντοπίστηκαν.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFolderButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Επιλογή Φακέλου...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="saveFolderDisplay">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="saveToTxtCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Αποθήκευση στιγμιότυπων όπου εντοπίστηκαν πρόσωπα σε αρχείο κειμένου.</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Παράμετροι</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Συντελεστής Κλίμακας (0.1 - 2.0):</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scaleFactorSpinBox">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>26</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ευαισθησία (0.01 - 0.99):</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="confidenceThresholdSpinBox">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>56</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>FaceDetectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>FaceDetectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
